Arrowgene.Ddon.Database/Sql/Core/DdonSqlDbCharacter.cs

```diff
@@ -274,6 +274,23 @@ private void QueryCharacterData(TCon conn, Character character)
                         item.Unk3 = GetByte(reader2, "unk3");
                         item.Color = GetByte(reader2, "color");
                         item.PlusValue = GetByte(reader2, "plus_value");
+                        item.EquipPoints = GetUInt32(reader2, "equip_points");
+                        ExecuteReader(conn, SqlSelectADDS,
```

Comment:
Since the table was commented out in the schema and is left out of the migration, this query will most likely return an error